sources/packages/backend/libs/services/src/queue/queue.module.ts

```diff
@@ -81,6 +81,9 @@ function getQueueModules(): BullModuleAsyncOptions[] {
       queueService: QueueService,
     ): Promise<BullModuleOptions> => {
       const queueConfig = await queueService.getQueueConfiguration(queue);
+      if (!queueConfig) {
+        return { prefix: undefined, defaultJobOptions: undefined };
```

Comment:
Why the queueConfig cannot be undefined/null?

Reply:
I just wanted the return data to be consistent with the format `{ prefix: ... , defaultJobOptions: ...}`, which is passed to [BullModule.registerQueueAsync(...getQueueModules())](https://github.com/bcgov/SIMS/blob/main/sources/packages/backend/libs/services/src/queue/queue.module.ts#L27). I tried returning undefined just now, and it also worked. Let me know if we want to return undefined instead @andrepestana-aot @andrewsignori-aot 🙂

Reply:
I do not see a reason to change this. Please note the below comment also.
Maybe I am missing something. Can you please elaborate on the motivation?
https://github.com/bcgov/SIMS/pull/3070/files#r1552604114

Reply:
I have a question here. Why do we need to change the queue configuration for the inactive queues (set them to undefined etc) if the queue itself is going to be obliterated.